packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -3,21 +3,25 @@ import classNames from 'classnames';
 import { getStatusBadgeConfig } from '@deriv/account';
```

Comment:
@heorhi-deriv please order the imports according to the [convention](https://github.com/binary-com/deriv-app/tree/master/docs/JavaScript#import-rules) followed 😄 .